packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -401,6 +401,7 @@ const TRANSFORMS = {
       installTime: metrics && toTimestamp(metrics.install_time),
       name_description: obj.name_description,
       name_label: obj.name_label,
+      notes: otherConfig['xo:notes'],
```

Comment:
Is it really necessary?

Reply:
Not strictly necessary, but easier to use I think.